paper.tex

```diff
@@ -439,32 +439,77 @@ \subsection*{Edge annotated recombination}
 }
 \end{figure}
 
+\section*{Relationship with other types of phylogenetic networks}
+
+The term \emph{phylogenetic network} has been broadly accepted to describe the most general 
+type of genealogical graph, defined by \citet{huson2010phylogenetic} as ``any graph used to 
+represent evolutionary relationships (either abstractly or explicitly) between a set of taxa 
+that labels some of its nodes" (although more narrow and context-dependent definitions can also 
+be found in the literature). Explicit rooted phylogenetic networks are rooted DAGs,
+with leaves labelled by the taxa (species, groups, individuals or sequences), and nodes classed 
+as either reticulation nodes (with in-degree $\geq 2$) or tree nodes. 
+
+Some sub-classes of explicit rooted phylogenetic networks are defined based on the evolutionary event that the
+reticulation nodes capture \citep{huson2010phylogenetic}. In \emph{reassortment} networks, used to 
+model the evolution of viruses, reticulation nodes correspond to exchange of 
+(non-recombining) genomic segments between viral particles inside a co-infected host. 
+In \emph{hybridisation} networks, reticulation nodes correspond to genetic material from two
+different species combining to create a hybrid. In \emph{recombination} networks, the reticulation nodes 
+correspond to recombination events (with leaves labelled by sequences and edges labelled by mutations). 
+The ARG is a subtype of a recombination network under this definition (restricting to one crossover 
+recombination per reticulation node).
+% AI: Not sure about this - do ARGs also encompass multiple crossover recombinations and gene
```

Comment:
Yes - this is a tricky issue. I think the original definitions were, but we're suggesting a more general definition here. Let's leave it for now though.